src/middlewares/validations/returnTripValidations.js

```diff
@@ -0,0 +1,51 @@
+import Joi from '@hapi/joi';
+
+export const validateReturnTrip = (req, res, next) => {
+  const schema = Joi.object({
+    origin: Joi.number()
+      .required()
+      .messages({
+        'number.base': res.__('Please fill your location'),
+        'any.required': res.__('Location is required')
+      }),
+    destination: Joi.number()
+      .required()
+      .messages({
+        'number.base': res.__('Please fill your destination'),
+        'any.required': res.__('Destination is required')
+      }),
+    from: Joi.date()
+      .required()
+      .messages({
+        'date.base': res.__('Please enter a valid date like YYY-MM-DD'),
+        'any.required': res.__('Start date is required')
+      }),
+    till: Joi.date()
+      .required()
+      .messages({
+        'date.base': res.__('Please enter a valid date like YYY-MM-DD'),
+        'any.required': res.__('End date is required')
+      }),
+    travelReasons: Joi.string()
+      .required()
+      .messages({
+        'string.empty': res.__('Please fill your travel reason'),
+        'any.required': res.__('Travel reason is required')
+      }),
+    accomodationId: Joi.number()
+      .required()
+      .messages({
+        'number.base': res.__('Please choose accomodation'),
+        'any.required': res.__('Accomodation is required')
+      }),
+    type: Joi.number()
+      .required()
+      .messages({
+        'number.base': res.__('Please choose type of trip'),
+        'any.required': res.__('Type is required')
+      })
```

Comment:
Remove type from validations since it's not required

Reply:
I am going to work on it

Reply:
solved!